### PR TITLE
Added PyLibFTDI 0.19.0

### PR DESCRIPTION
**Pull request by wvreeven**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/pylibftdi/LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,23 @@
+Copyright (c) 2010-2020 Ben Bass
```
Comment (dopplershift) [inline recipes/pylibftdi/LICENSE.txt:1]: This file is already included with the PyPI tarball so you shouldn't include it manually in the recipe.

**Review by dopplershift — CHANGES_REQUESTED** [review]
Just remove the unneeded file and this is good to go.

File: recipes/pylibftdi/LICENSE.txt (hunk 1 above)
Comment (wvreeven) [reply recipes/pylibftdi/LICENSE.txt]: Thanks for the quick review. I removed the file and the reference to it in `meta.yaml`.

Comment (wvreeven) [reply recipes/pylibftdi/LICENSE.txt]: And now I get the conda-forge-linting error "license_file entry is missing, but is required". How can I solve that?

Comment (dopplershift) [reply recipes/pylibftdi/LICENSE.txt]: Put back `license_file: LICENSE.txt` (and only that one line) 😉

**Review by dopplershift — APPROVED** [review]
Pending CI.

**Pull request merged**